build.sh

```diff
@@ -44,35 +43,21 @@ else
     echo "Patching Complete"
 fi
 
-if [ -d "${LIBYAML_DIR}" ]; then
+if [ -d "${UT_CONTROL_LIB_DIR}" ]; then
     echo "Framework libyaml already exists"
 else
     pushd ${FRAMEWORK_DIR} > /dev/null
-    echo "Clone libfyaml in ${LIBYAML_DIR}"
-    wget https://github.com/pantoniou/libfyaml/archive/refs/heads/master.zip --no-check-certificate
-    unzip master.zip
-
-    echo "Patching Framework [${PWD}]"
-    cp ../src/libyaml/patches/CorrectWarningsAndBuildIssuesInLibYaml.patch  .
-    patch -i CorrectWarningsAndBuildIssuesInLibYaml.patch -p0
-    echo "Patching Complete"
-
-#    ./bootstrap.sh
-#    ./configure --prefix=${LIBYAML_DIR}
-#    make
-    popd > /dev/null
+    if [ "$1" != "no_ut_control" ]; then
+        echo "Clone ut_control in ${UT_CONTROL_LIB_DIR}"
+        #TODO: change the git url to correct one
+        git clone https://github.com/rdkcentral/ut-control.git -b feature/gh20-control-plane-part2-librarycreation
+        #clone the external libraries
+        cd ut-control
+        ./install.sh
```

Comment:
it's not `install.sh` because your not installing anything your actually configuring the library for building you should cal lit  `configure.sh`